### DISCUSSION TITLE
Is it good practice to use TranslatableMessage in addFlash()?

### DISCUSSION THREAD

**Discussion by freiondrej-lokalise** [symfony/symfony#51022] [Q&A]
Hello,

I really like how neat TranslatableMessage is. However, I'm having second thoughts whether passing it to addFlash() is a good practice. I'll be grateful for any inputs 🙏

**Comment by yun548** [comment]
Yes, it sounds perfectly fine to do so, what are you having second thoughts about ?

**Reply by freiondrej-lokalise** [reply]
@yun548 thank you for your blazing fast answer! 🙇 I was thinking about how the object gets serialized in the session and the potential overhead this might create.

**Reply by yun548** [reply]
I reckon it is still better than injecting the translator service in multiple parts of your application
